src/languages/en.js

```diff
@@ -282,6 +282,7 @@ export default {
         addPayPalAccount: 'Add PayPal account',
         editPayPalAccount: 'Update PayPal account',
         growlMessageOnSave: 'Your PayPal username was successfully added',
+        formatError: 'Invalid PayPal.me username',
```

Comment:
NAB: Sorry for being Nit-picky. but this does not sound friendly. You can ask this on Slack.

Reply:
I've asked, lets see if we get any response. Any suggestions from your side?

Reply:
I am not really good with content but this is not really blocking this so if you don't get any feedback, then it's fine.

Reply:
Okay got it. Thanks for the help.